liferay-echo/liferay-portal
# LPS-168579 Add new SourceFormat Rule

Motivation
=======
Currently `clay:dropdown-actions` taglib can be used without any label, breaking accessibility: https://issues.liferay.com/browse/LPS-168579

Proposed Solution
========
We have added a new SF rule that checks if any of "aria-label", "label" or "title" attributes are present, and giving an error if this is not the case.

Steps to Verify
========
1. Use clay:dropdown-actions somewhere in portal without any label
1. Try running SF

Alternative Solutions that were discarded (if applies)
========
We considered checking this in the taglib itself, but currently there is no easy way to check if "some of a list of attributes is present". That's why we have added the SF rule.

cc @ling-alan-huang
